website/docs/constrainttemplates.md

```diff
@@ -0,0 +1,152 @@
+---
+id: constrainttemplates
```

Comment:
Can you add a link to this page from https://open-policy-agent.github.io/gatekeeper/website/docs/howto#constraint-templates

Reply:
done